vc/vc.go

```diff
@@ -32,6 +32,15 @@ type DataModel[T CredentialSubject] struct {
 	IssuanceDate      string             `json:"issuanceDate"`               // https://www.w3.org/TR/vc-data-model/#issuance-date
 	ExpirationDate    string             `json:"expirationDate,omitempty"`   // https://www.w3.org/TR/vc-data-model/#expiration
 	CredentialSchema  []CredentialSchema `json:"credentialSchema,omitempty"` // https://www.w3.org/TR/vc-data-model-2.0/#data-schemas
+	Evidence          []Evidence         `json:"evidence,omitempty"`         // https://www.w3.org/TR/vc-data-model/#evidence
+}
+
+// Evidence represents the evidence property of a Verifiable Credential.
+type Evidence struct {
+	ID          string `json:"id,omitempty"`
+	Type        string `json:"type,omitempty"`
+	// todo is `AdditionalFields` the right name?
```

Comment:
paging @mistermoe or @decentralgabe on whether this is the right approach for https://github.com/TBD54566975/web5-go/issues/79

Reply:
this won't work since go is strongly typed, we'll have to add specific evidence models when we know about them. I would recommend keeping evidence as an `any`

Reply:
@decentralgabe we can write a custom marshal / unmarshal yeah? That's what we did for JWT as well

Reply:
we can